lib/usbmux/index.js

```diff
@@ -101,10 +121,12 @@ class Usbmux extends BaseServiceSocket {
   }
 
   _receivePlistPromise (timeout = 5000) {
-    return new B((resolve, reject) => {
-      this._decoder.once('data', resolve);
+    const tag = this._tag++;
```

Comment:
can there be a mathematical overflow on this tag?

Reply:
That would be impressive. I love you how you see stuff :D